### PR TITLE
[HxButton] To be able to set different icon color than Button text #466

### PR DESCRIPTION
**Pull request by Harvey1214**

### CODE REVIEW

**Comment by rqx110** [comment]
In this way, how we can set different icon color for every button?

**Comment by rqx110** [comment]
hi @Harvey1214 , it is better provide a sample in doc to show how to change the icon color.

And, it looks like you have forgot to modify when icon in end placement:
https://github.com/havit/Havit.Blazor/blob/e33210a5cc0a6906c6b2e6fefeb7d68d62c6882f/Havit.Blazor.Components.Web.Bootstrap/Buttons/HxButton.razor#L51

**Comment by Harvey1214** [comment]
@rqx110 I agree, the user will usually want to set the color to one of the theme colors, therefore the approach with the `IconCssClass` parameter is more suitable. Thank you for the recommendation.

File: Havit.Blazor.Components.Web.Bootstrap/wwwroot/defaults.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	--hx-autosuggest-dropdown-menu-height:				300px;	
 	/* HxButton */
 	--hx-button-space-between-text-icon:				0.25rem;
+	--hx-button-icon-color:								unset;
```
Comment (rqx110) [inline Havit.Blazor.Components.Web.Bootstrap/wwwroot/defaults.css:11]: not used any more. remove this in doc also

**Review by hakenr — APPROVED** [review]
lgtm

**Pull request merged**